Sources/YNetwork/NetworkManager/Progress/FileProgress.swift

```diff
@@ -15,6 +15,8 @@ public typealias Percentage = Double
 ///
 /// Guaranteed to be called back on the main thread
 public typealias ProgressHandler = (Percentage) -> Void
+public typealias CompletionHandler = (Data) -> Void
```

Comment:
need to provide doc for the public declaration

Reply:
Added doc comment